src/dev/impl/DevToys/Helpers/CertificateHelper.cs

```diff
@@ -68,6 +70,36 @@ internal static bool TryDecodeCertificate(string input, string? password, out st
                 return false;
             }
 
+            // Try to decode the X.509 extensions. Put this is a separate try/catch because it's not critical.
+            if (certificate.Extensions.OfType<X509Extension>().Any())
+            {
```

Comment:
Can we please add unit tests for this block?

Reply:
7cefd8127cf8b75c8d8fa8e9a685ecaef98f32ed